Docs update (#406)
elastic/apm-agent-android: docs/reference/configuration.md

# Configuration [configuration]

> 

Reviewer: Could we please add some words here as well to give an introduction, short summary of the rest of the page?

# Configuration [configuration]

> This page is meant to complement the [Getting started](getting-started.md) one. Here you'll find a comprehensive list of all the configurable parameters available for the agent, including those you can set during initialization and those you can adjust dynamically afterward.

Reviewer 2: ```suggestion
This page includes a comprehensive list of all the configurable parameters available for the agent, including those you can set during initialization and those you can adjust dynamically afterward.

**Just getting started?** Start with [Getting started](getting-started.md). 
```
